identity/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## 0.1.2 - unreleased
 
+- Clarified deprecation notes on `Keypair` and `PublicKey`
```

Comment:
```suggestion
```

These deprecation notes aren't released yet, hence we don't need to document that they are changing :)